### PR TITLE
add submissions email

### PR DESCRIPTION
**Pull request by lauterb**

### CODE REVIEW

**Comment by lauterb** [comment]
Let me know if we should tweak the wording

File: CorpusSearch/ClientApp/src/components/Home.tsx [hunk 1]
```diff
@@ -159,7 +159,7 @@ export const Home = () => {
                         </span>
                     </>
                 }
-                <span style={{display: "inline", marginTop: "2em"}}>Support our revitalisation efforts by <a href={"/MailingList"}>signing up for our mailing list</a>. We'll email once in a while with updates to the corpus & other projects</span>
+                <span style={{display: "inline", marginTop: "2em"}}>Support our revitalisation efforts by <a href={"/MailingList"}>signing up for our mailing list</a>. We'll email once in a while with updates to the corpus & other projects. Send submissions to <a href="mailto:corpus-submissions@gaelg.im">corpus-submissions@gaelg.im</a></span>
```
Comment (david-allison) [inline CorpusSearch/ClientApp/src/components/Home.tsx:162]: Thoughts? We definitely want the line-break. Implementer's choice for the wording

```suggestion
                <span style={{display: "inline", marginTop: "2em"}}>Support our revitalisation efforts by <a href={"/MailingList"}>signing up for our mailing list</a>. We'll email once in a while with updates to the corpus & other projects.</span>
                <br/>
                <span>If we're missing anything, please let us know at <a href="mailto:corpus-submissions@gaelg.im">corpus-submissions@gaelg.im</a>.</span>
```

Screenshot 2024-03-30 at 20 15 31

----

My main goal here is to ensure that we don't have any 'unknown unknown' documents. The secondary goal is to reduce the effort to the minimum possible, we're more likely to have people email us if they feel they won't need to reformat/transcribe/translate a document, and we can add this to a work queue

Possibly: `submissions` -> `tips`/`leads`

Comment (lauterb) [reply CorpusSearch/ClientApp/src/components/Home.tsx:162]: Makes sense. Your change is definitely an improvement. "Submission" sounded more like a contest to me. I actually had a line break to start but took it out for some reason.

Comment (david-allison) [reply CorpusSearch/ClientApp/src/components/Home.tsx:162]: When you're free, feel free to push/force push to your `mailto` branch and it'll update here

[force push to PR branches strongly encouraged, but I can squash/rebase on my end]

**Review by lauterb — COMMENTED** [review]
Looks good, thanks!

**Comment by lauterb** [comment]
I squashed our commits into one on the branch and forced pushed.

**Comment by lauterb** [comment]
Unit test taking longer than usual. Maybe it's that time of the evening...the hour of the wolf or something.

**Comment by lauterb** [comment]
It's the post-run/checkout step where it is lingering

File: CorpusSearch/ClientApp/src/components/Home.tsx (hunk 1 above)
Comment (lauterb) [reply CorpusSearch/ClientApp/src/components/Home.tsx]: Heh, I didn't even notice your comment until after I squashed and force pushed.

**Review by david-allison — APPROVED** [review]
Looks great, thanks!

**Pull request merged**